BPA/Ch.07/cjs_mnl_ran.stan

```diff
@@ -77,25 +80,25 @@ generated quantities {
   real fit;
   real fit_new;
 
-  sigma2 <- square(sigma);
+  sigma2 = square(sigma);
 
   // Temporal variance on real scale
-  sigma2_real <- sigma2 * square(mean_phi) * square(1 - mean_phi);
+  sigma2_real = sigma2 * square(mean_phi) * square(1 - mean_phi);
```

Comment:
It's more efficient to reduce the overall number of operations that go onto the expression graph, so by that principle, `square(a * b)` is more economical than `square(a) * square(b)`.  The first only has two multiplications, whereas the latter has three.  It's no big deal, especially in generated quantities where there's no autodiff overhead.   So if you think the way you have it is clearer, please leave it that way.